### DISCUSSION TITLE
Modern Normalize

### DISCUSSION THREAD

**Discussion by luukdv** [tailwindlabs/tailwindcss#2563] [Ideas]
Browsers continue to evolve, but the last commit to Normalize was from 2018. Have you considered switching to https://github.com/sindresorhus/modern-normalize for Preflight? It's smaller (doesn't matter that much, but still), improved and up-to-date. Let me know if I can help.

**Comment by adamwathan** [comment]
Exploring different options for this right now actually:

https://twitter.com/adamwathan/status/1316435538900127751?s=21

I like modern-normalize but to switch to that we’d have to completely drop IE11 compatibility, which I’m not completely opposed to but is a bigger decision.
